GitUI/Translation/English.xlf

```diff
@@ -6562,15 +6562,15 @@ Nothing to rebase.</source>
         <target />
       </trans-unit>
       <trans-unit id="label1.Text">
-        <source>Less recent repositories</source>
+        <source>Recent repositories</source>
         <target />
       </trans-unit>
       <trans-unit id="lblRecentRepositoriesHistorySize.Text">
         <source>Recent repositories history size</source>
         <target />
       </trans-unit>
       <trans-unit id="maxRecentRepositories.Text">
-        <source>Maximum number of most recent repositories</source>
+        <source>Maximum number of pinned recent repositories</source>
```

Comment:
Should be "Maximum number of pinned repositories"

Reply:
This was intentional: The description is intended to differentiate pinned from favorites

Reply:
This made me question the difference between the pinned and favourites. I do feel that there's a significant overlap between these two, and a lot of added complexity. If we look at VS or other IDEs or productivity apps - most of those have a notion of "recent"s, and some have notion of "pinned" projects/files/etc.

Reply:
I say the GE repo favorites is a good name, there are categories and such.
In general, Pinned is similar to Favorites but you only select a certain items, no categories.

Most recent is in my opinion a better name than Pinned, as this is not something you select yourself. As both you and @pmiossec suggested the name so I agreed, to have the change ready for the alpha.
Maybe *Top recent* is better name?
I can just revert the renaming to Pinned too if you prefer that.